src/include/tracer.h

```diff
@@ -0,0 +1,64 @@
+#ifndef TRACER_H_
```

Comment:
I see there are two files src/common/jaegerTracer and this one, can you move them to src/common/JaegerTracer{.h/cc}